frontend/src/components/GAlertBanner.vue

```diff
@@ -57,6 +58,8 @@ const props = defineProps({
 })
 
 const { type, color } = toRefs(props)
+const localStorageStore = useLocalStorageStore()
+const hiddenMessages = toRef(localStorageStore, 'hiddenMessages')
```

Comment:
```js
const permanentlyHidden = computed({
  get () {
    if (props.identifier) {
      return hiddenMessages.value[props.identifier] === true
    }
    return false
  },
  set (value) {
    if (props.identifier) {
      hiddenMessages.value[props.identifier] = value
    }
  },
})
```
```
GAlertBanner.vue:71 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'cfg.4be0e4219e7ecb2bcc11fc2b18b25f37')
    at ReactiveEffect.get [as fn] (GAlertBanner.vue:71:34)
    at ReactiveEffect.run (reactivity.esm-bundler.js:178:19)
    at get value [as value] (reactivity.esm-bundler.js:1147:33)
    at ReactiveEffect.get [as fn] (GAlertBanner.vue:83:51)
    at ReactiveEffect.run (reactivity.esm-bundler.js:178:19)
    at get value [as value] (reactivity.esm-bundler.js:1147:33)
    at unref (reactivity.esm-bundler.js:1026:29)
    at Object.get (reactivity.esm-bundler.js:1032:35)
    at Proxy._sfc_render (GAlertBanner.vue:9:14)
    at renderComponentRoot (runtime-core.esm-bundler.js:816:16)
```

as the default for hiddenMessages  is gone, hiddenMessages  is null. 
You can reproduce it by setting an alert in your frontend config, e.g.

```yaml
frontend:
  alert:
    type: warning
    message: foo
```

Reply:
✔️